diff --git a/dist/theme.js b/dist/theme.js
index f73b985c0a4f0752ea7e3c949766bea603e0d7c3..cef0eb2049d4d234c062e0f1d2733cf8e5e9d5e4 100644
--- a/dist/theme.js
+++ b/dist/theme.js
@@ -141,6 +141,11 @@ class DocusaurusTheme extends theme_1.MarkdownTheme {
     }
     getTitle(page) {
         const readmeTitle = this.readmeTitle || page.project.name;
+
+        if(this.readmeTitle){
+            return readmeTitle;
+        }
+
         if (page.url === this.entryDocument && page.url !== page.project.url) {
             return readmeTitle;
         }